### PR TITLE
Removed bundle as in option the description of pkgtype since it is no…

### PR DESCRIPTION
**Pull request by grahampugh**
… longer valid

### CODE REVIEW

File: Code/autopkgserver/autopkgserver [hunk 1]
```diff
@@ -46,7 +46,7 @@ Request format:
 
     pkgname         The file or bundle name of the pkg, without pkg extension.
```
Comment (gregneagle) [inline Code/autopkgserver/autopkgserver]: Might want to remove other occurrences of "bundle", like here

File: Code/autopkgserver/autopkgserver [hunk 2]
```diff
@@ -46,7 +46,7 @@ Request format:
...
 
-    pkgtype         The package type, "flat" or "bundle".
+    pkgtype         The package type. "flat" is the only valid value.
```
Comment (gregneagle) [inline Code/autopkgserver/autopkgserver]: Maybe `"flat" is currently the only valid value.` to deflect questions about why pkgtype _exists_ if it has only one valid value...

**Comment by grahampugh** [comment]
I've removed the other references in `autopkgserver` and updated the commit. 

There are other references that may be relevant in `packager.py`, `PkgPayloadUnpacker.py`, `PkgInfoCreator.py` and `PkgExtractor.py`, but I'm not confident enough to know whether these are still required for legacy reasons.

**Comment by gregneagle** [comment]
"There are other references that may be relevant in packager.py, PkgPayloadUnpacker.py, PkgInfoCreator.py and PkgExtractor.py, but I'm not confident enough to know whether these are still required for legacy reasons."

Certainly there may still be bundle-style packages that AutoPkg may have to work with, even if it doesn't _create_ them. Adobe CCP output is bundle-style packages, for example.

**Pull request merged**